src/base/jsonwrapper.lua

```diff
@@ -0,0 +1,17 @@
+--
+-- jsonwrapper.lua
+-- Provides JSON encoding and decoding API by wrapping a third-party JSON library
+-- Copyright (c) 2017 Jason Perkins and the Premake project
+--
+
+	local implementation = dofile('json.lua')
+
+	json = {}
```

Comment:
I like this wrapper idea... just think it should be in the "premake.json" table, rather then a global json table...